ros2pkg/ros2pkg/api/create.py

```diff
@@ -89,180 +203,52 @@ def create_package_environment(package, destination_directory):
-        'package_name': package.name,
-    }
-    _create_template_file(
-        'cpp/main.cpp.em',
-        source_directory,
-        cpp_node_name + '.cpp',
-        cpp_node_config)
-
-
-def populate_cpp_library(package, source_directory, include_directory, cpp_library_name):
-    class_name = cpp_library_name.replace('_', ' ').title()
-    class_name = ''.join(x for x in class_name if not x.isspace())
-    cpp_header_config = {
-        'package_name': package.name,
-        'library_name': cpp_library_name,
-        'class_name': class_name,
-    }
-    _create_template_file(
-        'cpp/header.hpp.em',
-        include_directory,
-        cpp_library_name + '.hpp',
-        cpp_header_config)
-
-    cpp_library_config = {
-        'package_name': package.name,
-        'library_name': cpp_library_name,
-        'class_name': class_name
-    }
-    _create_template_file(
-        'cpp/library.cpp.em',
-        source_directory,
-        cpp_library_name + '.cpp',
-        cpp_library_config)
-
-    visibility_config = {
-        'package_name': package.name.upper(),
-    }
-    _create_template_file(
-        'cpp/visibility_control.h.em',
-        include_directory,
-        'visibility_control.h',
-        visibility_config)
+    return package_directory
+
+
+def display_create_package_environment_message(args, package):
+    """
+    Print stdout message describing the package details to be created.
+
+    :param args: Commandline args wrapped by package_create_arguments object.
+    :param package: Package details.
+    """
+    print('create package')
+    print('  package name:', package.name)
+    print('  destination directory:', os.path.abspath(args.destination_directory))
+    print('  package format:', package.package_format)
+    print('  version:', package.version)
+    print('  description:', package.description)
+    print('  maintainer:', [str(maintainer) for maintainer in package.maintainers])
+    print('  licenses:', package.licenses)
+    print('  build type:', package.get_build_type())
+    print('  dependencies:', [str(dependency) for dependency in package.build_depends])
+    if args.node_name:
+        print('  node_name:', args.node_name)
+    if args.library_name:
+        print('  library_name:', args.library_name)
+
+
+def valid_package_directory(package_path):
+    """
+    Determine if a package directory can be created.
+
+    :param package_path: Directory path to be created.
+    :return result: True indicates the package_path can be created.
+                    otherwise return false and error message.
+    :return err: Error message if package_path already exists.
+    """
+    result = True
+    err = None
+    if os.path.exists(package_path):
+        result = False
+        err = '\nAborted!\nThe directory already exists: ' + package_path + '.\nEither ' + \
+              'remove the directory or choose a different destination directory or package name.'
+    return result, err
```

Comment:
```suggestion
    return os.path.exists(package_path)
```
error handling can be done when this function is called.